solutions/baseline/variables.tf

```diff
@@ -0,0 +1,297 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API key."
+  sensitive   = true
+}
+
+variable "prefix" {
+  type        = string
+  description = "The prefix for resources created by this solution."
+  default     = null
+}
+
+##############################################################################
+# Cluster variables
+##############################################################################
+
+variable "cluster_id" {
+  type        = string
+  description = "The ID of the cluster to deploy the agents in."
+}
+
+variable "cluster_resource_group_id" {
+  type        = string
+  description = "The resource group ID of the cluster."
+}
+
+variable "cluster_config_endpoint_type" {
+  description = "Specify the type of endpoint to use to access the cluster configuration. Possible values: `default`, `private`, `vpe`, `link`. The `default` value uses the default endpoint of the cluster."
+  type        = string
+  default     = "private"
+  nullable    = false # use default if null is passed in
+  validation {
+    error_message = "The specified endpoint type is not valid. Specify one of the following types of endpoints: `default`, `private`, `vpe`, or `link`."
+    condition     = contains(["default", "private", "vpe", "link"], var.cluster_config_endpoint_type)
+  }
+}
+
+variable "is_vpc_cluster" {
+  type        = bool
+  description = "Specify true if the target cluster for the DA is a VPC cluster, false if it is classic cluster."
+  default     = true
+}
+
+variable "cluster_data" {
```

Comment:
Oh man, we cant use this. I think you copied this from Daniels POC dev branch?? All we need is the cluster Id which you already ask for. We dont need anything else in this map